### ISSUE TITLE
This seems dead

### ISSUE THREAD

**Issue by alexa-v2** [8bitPit/NiagaraWebsite#1]
There has been no activity on this in 7 months

Is it dead?

If you tell me want you want, I have some experience with designing websites

**Comment by Dadi-B** [comment]
Yeah I think the same... Even if 10 months have passed I tried contacting Peter via mail, but with a bot reply... I think I'll make a pull request to the project in the following weeks. I'm probably going to try to make as minimalistic as possible.

**Comment by 8bitPit** [comment]
I'm so sorry for not having responded to your messages on time. Thanks a lot for offering your help @alexa-v2 and thanks for your pr @Dadi-B, I'm humbled very much. Over the last months I've got familiar with the Vue.js framework, that might be overkill for just a "landing page", but I love the smoothness web apps can offer. I'll try to work on the website later this summer, I'll let you know when I start, afterwards we can build the web page together if you'd like to 😁.  
Thanks again guys.
